src/main/resources/db/migration/V4_3__update_behandler_kontor_herid.sql

```diff
@@ -0,0 +1,9 @@
+UPDATE BEHANDLER_KONTOR SET
+her_id='171911',
+orgnummer='927561867',
+navn='PSYKIATER SIGRID S. JOHNSEN',
+adresse='Starvhusgaten 2A',
+postnummer='5014',
+poststed='BERGEN',
```

Comment:
Føles litt u-nice å ha i gitloggen og i koden, men det er vel offentlig info kanskje? Er HER-ID for en lege sensitivt?

Reply:
Ja, oppfatter dette som offentlig info: vil anta at her-id er på linje med hpr-id og det er offentlig.